packages/venia-concept/src/components/Modal/modal.js

```diff
@@ -0,0 +1,6 @@
+import { createPortal } from 'react-dom';
+
+const Modal = ({ children }) =>
+    createPortal(children, document.getElementById('root'));
```

Comment:
This is so simple—I love it. Two thoughts come to mind here:

- Rather than hardcoding the target to `#root`, we might consider accepting an element as a prop, with `#root` being the default
- I wonder if we can memoize the `document.getElementById()` call with `useMemo()`

Reply:
Great suggestions. Element id prop and memoization has been added in the latest update.

Reply:
I had something *slightly* different in mind for the element prop—I wanted to cover a case in which you have an element stored as a ref and don't need to query the document—so I've pushed a small change. Overall this is just what I had in mind though. 👍